include/intrusive_graph.h

```diff
@@ -36,6 +36,7 @@ namespace detail {
 
 		auto& front() const { return *m_first; }
 		bool empty() const { return m_first == m_last; }
+		size_t size() const { return m_last - m_first; }
```

Comment:
Since this type has so far been well-defined for any iterator, `size()` should use `std::distance` instead of `operator-`.

Reply:
Done!